lib/rules/function-url-quotes/__tests__/index.js

```diff
@@ -710,6 +710,148 @@ testRule({
 	],
 });
 
+testRule({
```

Comment:
[suggest] Can you add `fix: true` to the existing `testRule` to reduce duplication?

https://github.com/stylelint/stylelint/blob/3b7a7d0a348494120d6f01d3d4acf949780d37bf/lib/rules/function-url-quotes/__tests__/index.js#L5-L7